rules/instance_network_interface_rules.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
```

Comment:
not sure what you mean from the second comment , like in the future that makes sense? Cause yeah that would be awesome if it was applicable in more ways.